### DISCUSSION TITLE
修改文件命名模板是否会重新下载

### DISCUSSION THREAD

**Discussion by zpq5935** [Johnserf-Seed/TikTokDownload#591] [Q&A]
修改 conf.init 的文件命名模板，
![image](https://github.com/Johnserf-Seed/TikTokDownload/assets/44501567/7006e1bd-e27d-43e5-8fe2-ba4a43f9cf60)

对于 文件检查（已下载则调过）来说 是不是算作未下载呢，我看好像就是根据文件名来的？

**Comment by Johnserf-Seed** [comment]
全局影响的，修改了就会重新下载

**Reply by zpq5935** [reply]
了解了 谢谢
